trex-operator/roles/app/templates/job.yml

```diff
@@ -9,9 +9,17 @@ spec:
     metadata:
       labels:
         example-cnf-type: pkt-gen-app
+{% if runtimeClassName is defined and runtimeClassName|length %}
```

Comment:
If we need this for trex, similarly to the other cases, we need to pass this variable when creating the CR, and if I'm not wrong, it's the TRexApp CR: https://github.com/rh-nfv-int/nfv-example-cnf-deploy/blob/master/roles/example-cnf-app/templates/trex-app-cr.yaml.j2